types/react-date-range/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "private": true,
+  "dependencies": {
+    "react-date-range": "^0.9.4"
```

Comment:
First, I do not think you do not need this. Second, this package does not have its own definition files; keep in mind that these tests do not really run, these are just compilation checks to make sure that the test code "compiles" with no errors.